packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1062,7 +1062,8 @@ const messages = {
         maxLength: "Maksimal lengde for tekstfeltet er nådd",
         error: "Feltet er påkrevd",
       },
-      bottomText: "Savner du en kategori? Gi vår moderator beskjed på moderator@ndla.no",
+      bottomText: "Savner du en kategori? Gi vår moderator beskjed på ",
```

Comment:
Om du gjør det slik kan du komme unna med å legge en parse() rundt tekstboksen i frontend.
```suggestion
      bottomText: "Savner du en kategori? Gi vår moderator beskjed på <a href=\"moderator@ndla.no\">moderator@ndla.no</a>",
```

Reply:
Må jeg ha parse() på den måten jeg har gjort det i frontend? [#1761](https://github.com/NDLANO/ndla-frontend/pull/1761)

Reply:
Nei der lager du jo en lenke eksplisitt med SafeLink. Bruk av parse() fra html-react-parser oversetter html fra tekststrengen.

Reply:
Ja, men blir det ryddigere å slenge hmtl elementer i stringen også parse det enn å bare splitte det og gjøre på denne måten? Vi har ikke mange eksempler der vi har html elementer i strenger i locale

Reply:
Vi har noen, men det er sikkert greiest slik du gjør det her.